kube/src/config/mod.rs

```diff
@@ -143,11 +143,10 @@ impl Config {
     /// and relies on you having the service account's token mounted,
     /// as well as having given the service account rbac access to do what you need.
     pub fn from_cluster_env() -> Result<Self> {
-        let cluster_url =
-            incluster_config::kube_server().ok_or_else(|| ConfigError::MissingInClusterVariables {
-                hostenv: incluster_config::SERVICE_HOSTENV,
-                portenv: incluster_config::SERVICE_PORTENV,
-            })?;
+        let cluster_url = incluster_config::kube_server().ok_or(ConfigError::MissingInClusterVariables {
+            hostenv: incluster_config::SERVICE_HOSTENV,
+            portenv: incluster_config::SERVICE_PORTENV,
+        })?;
```

Comment:
hah, i think i usually change towards or_else because of clippy, has this changed now?

Reply:
i guess they are constants. i see the new ``#[warn(clippy::unnecessary_lazy_evaluations)]` on by default`